Dockerfile

```diff
@@ -6,8 +6,12 @@ COPY . /usr/src/app
 
 RUN apt-get --yes update \
     && apt-get --yes upgrade \
-    && pip3 install pipenv \
-    && pipenv install --system --deploy --dev
+    && pip3 install pipenv
+
+RUN pip3 install mysqlclient
+RUN pipenv install --system --deploy --dev
```

Comment:
the pip3 install here is a temporary fix in order to move on from pipenv throwing errors when trying to install mysqlclient.  This will be reversed when time permits before the end of the semester.

Reply:
@nlMeminger Argh. This is annoying. Same thing on my machine too with the dependency. It must not be getting installed correctly. You're right, we should revisit later but I agree to prioritize getting a dev environment done first.